front/src/modules/trainschedule/components/TimetableV2/TimetableV2.tsx

```diff
@@ -478,34 +149,25 @@ const TimetableV2 = ({
         })}
       >
         {trainsDurationsIntervals &&
-          trainsList
-            .sort((trainA, trainB) => trainA.departure_time - trainB.departure_time)
-            .map((train: ScheduledTrain, idx: number) => (
-              <TimetableTrainCard
-                idx={idx}
-                isSelectable={multiselectOn}
-                isInSelection={selectedTrainIds.includes(train.id)}
-                toggleTrainSelection={toggleTrainSelection}
-                train={train}
-                intervalPosition={valueToInterval(train.duration, trainsDurationsIntervals)}
-                key={`timetable-train-card-${train.id}-${train.path_id}`}
-                isSelected={infraState === 'CACHED' && selectedTrainId === train.id}
-                isModified={trainScheduleIDsToModify.includes(train.id)}
-                projectionPathIsUsed={
-                  infraState === 'CACHED' &&
-                  !!selectedProjection &&
-                  selectedProjection.id === train.id
-                }
-                refetchTimetable={refetchTimetable}
-                setDisplayTrainScheduleManagement={setDisplayTrainScheduleManagement}
-                setTrainResultsToFetch={setTrainResultsToFetch}
-                simulation={simulation}
-                selectedProjection={selectedProjection}
-              />
-            ))}
+          trainSchedulesDetails.map((train: TrainScheduleWithDetails, idx: number) => (
+            <TimetableTrainCardV2
+              idx={idx}
+              isSelectable={multiselectOn}
+              isInSelection={selectedTrainIds.includes(train.id)}
+              toggleTrainSelection={toggleTrainSelection}
```

Comment:
Do you think it's a better name ?
```suggestion
              handleSelectTrain={handleSelectTrain}
```

Reply:
Yes I agree, it's not really a toggle